plugins/homepage/server/__init__.py

```diff
@@ -21,8 +21,10 @@
 from girder.api import access
 from girder.api.describe import Description, describeRoute
 from girder.api.rest import Resource
+from girder.utility.model_importer import ModelImporter
 
 KEY = 'homepage.markdown'
+NAME = 'Homepage Assets'
```

Comment:
These are typically in a `constants.py` file, as members of a class. It's probably not as big of a deal for a plugin this size.